packages/advanced-logic/specs/payment-network-any-declarative-0.1.0.md

```diff
@@ -10,9 +10,10 @@ Prerequisite: Having read the advanced logic specification (see [here](/packages
 ## Description
```

Comment:
@vrolland, regarding your comment about the file name, I believe it should not contain a version at all, but include one at the top of the file maybe